tests/dat/manifest_data_compose_actions_for_limits.yaml

```diff
@@ -21,7 +21,6 @@ packages:
       hello1:
         function: ../src/integration/helloworld/actions/hello.js
         limits:
-          timeout: 1
           timeout: 600000
```

Comment:
Should this be 60000 ?

Reply:
Not sure. I didn't change that value. Additionally, that test file is used in parsing tests so the values may not matter much.

Reply:
The value was not intended to be 60 seconds... just testing input matches output against some larger (max) value. Here 10 minutes...